sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/PartitionBasedLoadBalancer.java

```diff
@@ -168,125 +168,120 @@ void loadBalance() {
      * current Event Processor should take on the responsibility of processing more partitions.
      */
     private Mono<Void> loadBalance(final Tuple2<Map<String, PartitionOwnership>, List<String>> tuple) {
-        return Mono.fromRunnable(() -> {
```

Comment:
As `claimOwnership()` and `renewOwnership()` will return `Mono<Void>` and we return them, here we remove the `Mono.Runnable`. Most of the changes are formatting, hide the whitespace can review more clearly.